libraries/radwordsprocessing/editing/find-and-replace/repalce-document-elements.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Replace Text with Document Elements
+page_title: Replace Text with Document Elements
+description: Learn how to find and replace text with various elements in a document using WordsProcessing.
+slug: radwordsprocessing-editing-replace-document-elements
+tags: find,search,replace
+published: True
+position: 6
+---
+
+# Replace Text with Document Elements
+
+**RadWordsProcessing** gives you the ability to search for a string in a [RadFlowDocument]({%slug radwordsprocessing-model-radflowdocument%}) instance and replace all matches with specified blocks or inlines. 
+
+You can search and replace text  using [RadFlowDocumentEditor]({%slug radwordsprocessing-editing-radflowdocumenteditor%}). This article lists the available methods and describes how you can use them.
+
+**RadFlowDocumentEditor** exposes the **ReaplaceText()** method to enable you to find all instances of a string. You can choose between several overloads that allow you to replace the text with one or more blocks or inlines. 
+
+> This functionality is available with R3 2021.
+
+
+## Replace Text with one or more inlines
```

Comment:
Fix casing to comply to the [style guide](https://testdocs.telerik.com/style-guide/titles-and-headings#capitalization)